conf/application.conf

```diff
@@ -45,7 +45,7 @@ play.http.session.sameSite: 'None'
 
 play.i18n.langs = ["en", "cy"]
 
-play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 localhost:12345 cdn.optimizely.com www.google-analytics.com www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com data:"
+play.filters.headers.contentSecurityPolicy= "'self' localhost:9000 localhost:9032 localhost:12345 www.google-analytics.com www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com data:"
```

Comment:
`default-source` will still be needed here otherwise there's no directive and play doesn't seem to fallback to a default.

Ideally, we'd switch to the new Play CSP Filter and Configuration so that we can do a more fine grained CSP - but we can get a separate story for that.

Without the `default-src` directive the CSP is not parsed as it thinks that `'self'` is the directive:
![image](https://user-images.githubusercontent.com/16954819/194530117-1ca06d3d-f73e-4e4b-948a-186925695b16.png)